pkg/apis/servicecatalog/types.go

```diff
@@ -145,10 +145,15 @@ type CommonServiceBrokerSpec struct {
 // ServiceClass allowed property names:
 //   name - the value set to [Cluster]ServiceClass.Name
 //   externalName - the value set to [Cluster]ServiceClass.Spec.ExternalName
+//   externalID - the value set to [Cluster]ServiceClass.Spec.externalID
+//   serviceBrokerName - the value set to [Cluster]ServiceClass.Spec.serviceBrokerName
```

Comment:
The key for this property should be spec.serviceBrokerName and spec.clusterServiceBrokerName, depending on whether it refers to a ServiceBroker or a ClusterServiceBroker.